globals.d.ts

```diff
@@ -1 +1 @@
-declare var fetchMock: any;
+declare let fetchMock: never;
```

Comment:
This is probably wrong. Should probably see if there are types for fetch mock instead.

Reply:
Agreed, I'll take a look at doing that in this pr.

Reply:
done in 98c2312